lib/voicebase/v2/client.rb

```diff
@@ -79,6 +133,22 @@ def get_media_progress(args = {}, headers = {})
         ), api_version)
       end
 
+      def delete_file(args = {}, headers = {})
+        url = if args[:media_id]
+                uri + "/media/#{args[:media_id]}"
+              else
+                raise ArgumentError, "Missing argument :media_id"
+              end
+
+        response = self.class.delete(
+            url,
```

Comment:
Not a strong reason to change... I think Rubymine defaults to some ugly choices though, haha!